drafts/celebrating-with-pictures.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Celebrating a birthday with pictures
+author: sébastiaan
```

Comment:
For this author to work you need to add info with the same key in _config.yml, see https://github.com/svthalia/blog/blob/713fd51bc5b3b17570c8c1f40884217630ffda2b/_config.yml#L4 from #3